packages/cmr-client/src/EarthdataLogin.ts

```diff
@@ -0,0 +1,202 @@
+// @ts-nocheck
```

Comment:
See above comment - why is our typescript semantic checking disabled?    This seems like a bad thing.

Reply:
Discussed in Slack. We'll issue a PR to main to correct this.